modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/CreationMenu.js

```diff
@@ -12,36 +12,53 @@
  * details.
  */
 
-import {ClayButtonWithIcon} from '@clayui/button';
+import ClayButton from '@clayui/button';
 import ClayDropDown from '@clayui/drop-down';
+import ClayIcon from '@clayui/icon';
+import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import React, {useContext, useState} from 'react';
 
 import FrontendDataSetContext from '../../FrontendDataSetContext';
 import {triggerAction} from '../../utils/actionItems/index';
 
-function CreationMenu({primaryItems}) {
+function CreationMenu({inEmptyState, primaryItems}) {
 	const frontendDataSetContext = useContext(FrontendDataSetContext);
 
 	const {loadData} = frontendDataSetContext;
 
 	const [active, setActive] = useState(false);
 
+	const labelNew = Liferay.Language.get('new');
```

Comment:
(nit, optional) SF rule in backend would be have a blank line after this line. Two sibling lines of code have a break line by default, omitting it only if they are doing something very similar (like several `sb.append`s for example). "Similarity" is a tad subjective, but is completely not applicable when something is defined in one line and used in another.